packages/search/lib/utils.js

```diff
@@ -6,9 +6,16 @@ const ES_INDEX_PREFIX = 'republik' // TODO: Put into config
  * @example "2018-05-24-17-05-23"
  * @return {String} [description]
  */
-const sanitizedDate =
+const getDateTime =
   () => new Date().toISOString().slice(0, -5).replace(/[^\d]/g, '-')
 
+/**
+ * @example "2018-05-24"
+ * @return {String} [description]
+ */
+const getDate =
+  () => new Date().toISOString().slice(0, -14).replace(/[^\d]/g, '-')
```

Comment:
you know we use [moment](https://github.com/orbiting/backends/blob/b2818c039254f5aac9dd6c166524fd018317b9c0/servers/republik/package.json#L29) in other places already don't you? Anyway I'm fine with this as well.

Reply:
No, I was not aware of that. Very good to now.